src/overlays/actors/ovl_Demo_Shd/z_demo_shd.c

```diff
@@ -29,16 +28,363 @@ const ActorInit Demo_Shd_InitVars = {
     (ActorFunc)DemoShd_Update,
     (ActorFunc)DemoShd_Draw,
 };
-*/
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Demo_Shd/DemoShd_SetupAction.s")
+u32 D_80991680[] = {
```

Comment:
These should be Gfx arrays I assume? And given their size, should they go into their own file?

Reply:
The things that get passed to gSPDisplayList are display lists, and those should be disasembled with gfxdis. you can use gfxdis to parse static dlists at a certain offset in the rom.

The dlists will then reference the other stuff as either textures or verities. It will be clear which is which from the dlists after they have been dissasembled. Textures can be typed as unk_type and vertices are Vtx and should be written using the macros (use vtxdis in tools).

And yeah id personally make a gfx file for this